content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/_cq_design_dialog/.content.xml

```diff
@@ -118,6 +118,42 @@
                         jcr:primaryType="nt:unstructured"
                         sling:resourceType="granite/ui/components/coral/foundation/include"
                         path="/mnt/overlay/cq/gui/components/authoring/dialog/style/tab_design/styletab"/>
+                    <hierarchy
```

Comment:
Adding a new tab is a breaking change (as components extending page `v2` might have already defined a tab with the same name, see https://github.com/adobe/aem-core-wcm-components/wiki/Versioning-Policies#dialogs

I think it would be best to create a `v3` for this enhancement.

Reply:
@gabrielwalt You may want to review the fields label and description